sites/fast-component-explorer/app/fast-components/configs/fast-flipper.definition.ts

```diff
@@ -0,0 +1,38 @@
+import { WebComponentDefinition } from "@microsoft/fast-tooling/dist/data-utilities/web-component";
+import { DataType } from "@microsoft/fast-tooling";
+import { FlipperDirection } from "@microsoft/fast-foundation";
+
+export const fastFlipperDefinition: WebComponentDefinition = {
+    version: 1,
+    tags: [
+        {
+            name: "fast-flipper",
+            description: "The FAST flipper element",
+            attributes: [
```

Comment:
are these definition files something that should live with the components themselves? I'm worried about keeping these in sync with component implementations.

Reply:
These will be autogenerated, I'm starting an investigation into a few packages that could achieve this and then I'll determine what tweaks may need to be made so I see this is a temporary solution.